### PR TITLE
[ID] Update Rules, Add COC and Abuse

### PR DESCRIPTION
**Pull request by rorre**

### CODE REVIEW

File: wiki/Contributor_Code_of_Conduct/id.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Kode Etik Contributor Covenant
+
+## Ikrar Kami
+
+Kami sebagai anggota, kontributor, dan pengelola berikrar untuk membuat partisipasi dalam komunitas kami terbebas dari pelecehan bagi semua orang, tanpa memandang usia, ukuran tubuh, disabilitas yang tampak maupun tidak, kelompok etnis, karakteristik seksual, identitas gender dan ekspresinya, tingkat pengalaman, pendidikan, status sosial ekonomi, kewarganegaraan, penampilan pribadi, ras, agama, atau identitas dan orientasi seksual.
+
+Kami berjanji untuk bertingkah dan berinteraksi dalam langkah komunitas yang terbuka, ramah, inklusif, dan sehat.
```
Comment (hikikuman) [inline wiki/Contributor_Code_of_Conduct/id.md:7]:
```suggestion
Kami berjanji untuk bertingkah dan berinteraksi dalam langkah komunitas yang terbuka, ramah, beragam, inklusif, dan sehat.
```
di en.md itu ada `diverse` sebelum inklusif. mungkin bisa bisa dimasukin itu kata `beragam`.

File: wiki/Contributor_Code_of_Conduct/id.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Standar Kami
+
+Contoh perilaku yang berkontribusi memberi dampak yang positif dalam komunitas kami meliputi:
+
+- Menunjukkan rasa empati dan kebaikan kepada orang lain
+- Menghormati opini, pandangan, dan pengalaman yang berbeda
+- Memberikan dan menerima secara sopan umpan balik yang konstruktif
+- Menerima tanggung jawab dan meminta maaf kepada yang terdampak dari kesalahan kita, dan belajar dari pengalaman
```
Comment (hikikuman) [inline wiki/Contributor_Code_of_Conduct/id.md:16]:
```suggestion
- Menerima tanggung jawab dan meminta maaf kepada mereka yang terdampak dari kesalahan kita, dan belajar dari pengalaman
```
ada kurang kata `mereka` disitu, agak ganjel juga kalo tanpa kata `mereka`. di en.md nya juga ada `those`

**Review by Nivalyx — COMMENTED** [review]
looking in the big picture the translation in general is alright as is i think...

it's worth noting tho that (iirc) the id wiki has used "kamu" instead of "anda" nearly all the time so it might be worth to review the instances of "anda" here

File: wiki/Contributor_Code_of_Conduct/id.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- Fokus terhadap apa yang terbaik tidak hanya untuk diri pribadi, tetapi untuk keseluruhan komunitas
+
+Contoh perilaku yang tidak dapat diterima meliputi:
+
+- Penggunaan bahasa atau citra seksual, dan sorotan bernada seksual atau rayuan dalam bentuk apapun
+- Mengejek, mencaci maki atau komentar yang menghina / merendahkan, dan serangan terhadap individual atau politik
```
Comment (hikikuman) [inline wiki/Contributor_Code_of_Conduct/id.md:22]:
```suggestion
- Komentar yang bersifat mengejek, mencaci maki atau menghina/merendahkan, dan serangan terhadap individual atau politik
```
komentarnya pindahin ke awal sama garis miringnya disatukan

**Comment by rorre** [comment]
> it's worth noting tho that (iirc) the id wiki has used "kamu" instead of "anda" nearly all the time so it might be worth to review the instances of "anda" here

Yes, I am fully aware of it. However the use of `kamu` is usually used where the wording is a bit more "laid back" or the page is often visited (such as Community Rules), while the use of `Anda` is used in a much more serious and formal condition.

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ tags:
 
 **Global Moderation Team** (sebelumnya dikenal sebagai Administrator, Tim Global Mod atau GreenBAT), sering disebut sebagai **GMT** adalah anggota dari komunitas yang berfokus terutama pada moderasi pemain dan isu-isu yang berkaitan dengan staf dan komunitas pada umumnya.
 
+**Semua Global Moderator bersumpah untuk menjunjung tinggi [Kode Etik Kontributor](/wiki/Contributor_Code_of_Conduct) sebagai tambahan dari [Peraturan Komunitas](/wiki/Rules).**
+
+Jika anda menemukan anggota Global Moderation Team bertindak tidak senonoh, harap segera kontak [abuse@ppy.sh](mailto:abuse@ppy.sh) dengan informasi sebanyak mungkin. Untuk informasi lebih lanjut, lihat [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).
```
Comment (hikikuman) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:14]:
```suggestion
Jika kamu menemukan anggota Global Moderation Team bertindak tidak senonoh, harap segera kontak [abuse@ppy.sh](mailto:abuse@ppy.sh) dengan informasi sebanyak mungkin. Untuk informasi lebih lanjut, lihat [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).
```
`anda` yang lain itu pake `kamu`. ada satu itu dibawahnya masih `anda`, itu salah ku ternyata blm ke ganti pas translate wiki ini :c

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 5]
```diff
@@ -9,6 +9,10 @@ tags:
...
+
 ## Peran dan Tanggung Jawab
 
 Global Moderation Team bertanggung jawab atas kesejahteraan obrolan dan forum, dan mengurus interaksi di dalam komunitas. Mereka harus didekati jika Anda memiliki pertanyaan yang berkaitan dengan aspek moderasi permainan, yang akan mencakup (namun tidak terbatas pada) masalah dengan obrolan (silences, pelecehan) and di forum (spam).
```
Comment (hikikuman) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:18]:
```suggestion
Global Moderation Team bertanggung jawab atas kesejahteraan obrolan dan forum, dan mengurus interaksi di dalam komunitas. Mereka harus didekati jika kamu memiliki pertanyaan yang berkaitan dengan aspek moderasi permainan, yang akan mencakup (namun tidak terbatas pada) masalah dengan obrolan (silences, pelecehan) and di forum (spam).
```

File: wiki/People/The_Team/Nomination_Assessment_Team/id.md [hunk 6]
```diff
@@ -9,6 +9,10 @@ tags:
 
 Anggota NAT dibedakan berdasarkan gelar dan nama pengguna mereka yang berwarna merah. Sama seperti [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) (*GMT*), mereka mempunyai izin moderasi di seluruh situs dan nama pengguna berwarna merah tua di kanal obrolan in-game.
 
+**Semua anggota Nomination Assessment Team bersumpah untuk menjunjung tinggi [Kode Etik Kontributor](/wiki/Contributor_Code_of_Conduct) sebagai tambahan dari [Peraturan Komunitas](/wiki/Rules).**
+
+Jika anda menemukan anggota Nomination Assessment Team bertindak tidak senonoh, harap segera kontak [abuse@ppy.sh](mailto:abuse@ppy.sh) dengan informasi sebanyak mungkin. Untuk informasi lebih lanjut, lihat [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).
```
Comment (hikikuman) [inline wiki/People/The_Team/Nomination_Assessment_Team/id.md:14]:
```suggestion
Jika kamu menemukan anggota Nomination Assessment Team bertindak tidak senonoh, harap segera kontak [abuse@ppy.sh](mailto:abuse@ppy.sh) dengan informasi sebanyak mungkin. Untuk informasi lebih lanjut, lihat [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).
```

**Comment by TicClick** [comment]
@hikikuman @Nivalyx is this good to fly?

**Comment by Nivalyx** [comment]
@TicClick yeah it should be good

**Review by TicClick — APPROVED** [review]

**Pull request merged**